client/dist/styles.css

```diff
@@ -1,53 +1,111 @@
 #app {
   background-color: lightgray;
   display: flex;
-  margin-left: auto;
-  margin-right: auto;
-}
 
-#imageModalImage #PanAndZoomImage-container{
-  margin: 0;
-  background: black;
-  position: fixed;
-  top: 0%;
-  left: 0%;
-  transition: all .2s ease;
-  /* padding: 100% */
 }
 
-#imageModalBG {
+#ImageModal{
+  z-index: 99 !important;
+  position: absolute;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  top: 5vh;
+  left: 5vw;
+  height: 90vh;
+  width: 90vw;
+  overflow: scroll;
+}
+.modalZoom {
+  z-index: 1;
+  position: absolute;
+  cursor: zoom-in;
+}
+.show {
+  opacity: 0;
+  transition: opacity 0.2s;
+  display: block;
+}
+.noShow {
+  opacity: 1;
+  display: block;
+  transition: opacity 0.2s;
+}
+.imageModalBG {
+  z-index: 0;
   position: fixed;
-  top: 0%;
   background-color: black;
-  opacity: 0.85;
-  padding-top: 50%;
+  opacity: 0.5;
   padding: 100%;
+  overflow: hidden;
+}
+#modalControls {
+  background-color: rgba(0, 0, 0, 0.5);
+  position: absolute;
+  display: flex;
+  bottom: 0px;
+  width: 40vw;
+  z-index: 2;
+  height: 3vh;
+}
+#modalControls button {
+  width: 6vw;
+  border: transparent;
+  background-color: gainsboro;
+  font-weight: bolder;
+  font-size: large;
+}
+.modalIndicator {
+  border: solid white 2px;
+  width: 0.7vw;
+  height: 0.7vw;
+  margin: 0.2vw;
+  border-radius: 50%;
+}
+.modalIndicator.selected {
+  background-color: white;
+  color: white;
+  opacity: 1;
+}
+#indicatorsContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%
 }
 
-#picked {
-  border: 3px solid rgb(246, 20, 65);
-  color: whitesmoke
+.pickerImageBG {
+  position: relative;
+  display: block;
+  margin: 10px;
```

Comment:
I noticed that you've tried to use relative units throughout your code, do you have to use absolute units in here?